src/app/course-page/course-search/course-search.component.ts

```diff
@@ -1,12 +1,14 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 
 @Component({
   selector: 'course-search',
   templateUrl: './course-search.component.html',
   styleUrls: ['./course-search.component.scss'],
 })
-export class CourseSearchComponent implements OnInit {
-  constructor() {}
+export class CourseSearchComponent {
+  public courseSearch: string;
 
-  ngOnInit(): void {}
+  public searchCourse(searchElement): void {
+    console.log(searchElement);
```

Comment:
you will need to emit this to the parent component so it can filter out courses

Reply:
done